### PR TITLE
Add docsite support

### PR DESCRIPTION
**Pull request by nventuro**
Part of https://github.com/OpenZeppelin/docs.openzeppelin.com/issues/47

I set up docs-utils and netlify build and split the readme into pages, following other projects. I didn't alter the content (though we may want to revise it at some point: it'd help to have a shorter overview, and go a bit more in-depth in a getting started guide).

### CODE REVIEW

File: docs/antora.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: network.js
+title: Network JS
+version: 0.0
```
Comment (frangio) [inline docs/antora.yml]: `oz-docs init` should've set this up correctly I think... Not sure what happened.

```suggestion
version: 0.2
```

File: docs/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* xref:index.adoc[Overview]
```
Comment (frangio) [inline docs/modules/ROOT/nav.adoc]: Are we using "Overview" or "Getting started"?

File: docs/antora.yml (hunk 1 above)
Comment (frangio) [reply docs/antora.yml]: https://github.com/OpenZeppelin/docs-utils/blob/dc7ce3006b6065cc4edebe53896644da5bd7dbec/oz-docs.js#L219-L229

File: docs/modules/ROOT/nav.adoc (hunk 2 above)
Comment (nventuro) [reply docs/modules/ROOT/nav.adoc]: Overview is the landing, with description, sample usage, and links.

Getting Started is an in-depth guide to using the thing. We don't have one for some of our packages (like test-helpers, which is is just overview, config and API ref).

I'll put together some guidelines to keep everything consistent.

File: docs/antora.yml (hunk 1 above)
Comment (nventuro) [reply docs/antora.yml]: :man_shrugging:

**Pull request merged**